velox/functions/prestosql/Rand.h

```diff
@@ -28,6 +28,33 @@ struct RandFunction {
     result = folly::Random::randDouble01();
     return true;
   }
+
+  FOLLY_ALWAYS_INLINE void call(int64_t& out, const int64_t& input) {
+    inputCheck(input);
```

Comment:
naming: method names should start with a verb: checkInput

naming: 'input' is a very generic name; consider changing to 'n' and 'checkBound'

```
checkBound(n);
```